unicode_variable_repr.py

```diff
@@ -142,5 +149,12 @@ def uc(hex):
     'C_L_R_T': C_L_R_T,
     'C_L_R_NT': C_L_R_NT,
     'B_L_U_NT': B_L_U_NT,
-    'B_L_R_NT': B_L_R_NT
+    'B_L_R_NT': B_L_R_NT,
+    'L_F_H_U_T':L_F_H_U_T
+    'L_F_H_R_T':L_F_H_R_T
+    'L_B_H_R_T':L_B_H_R_T
+    'L_F_M_U_T':L_F_M_U_T
+    'L_F_M_R_T':L_F_M_R_T
+    'L_B_M_R_T':L_B_M_R_T
+    'L_C_L_U_NT':L_C_L_U_NT
```

Comment:
do these two "L"s mean the same thing? I'm guessing not. In that case, we might need different identifiers